osf/models/registrations.py

```diff
@@ -432,28 +440,71 @@ class DraftRegistrationLog(ObjectIDMixin, BaseModel):
     action = models.CharField(max_length=255)
     draft = models.ForeignKey('DraftRegistration', related_name='logs',
                               null=True, blank=True, on_delete=models.CASCADE)
-    user = models.ForeignKey('OSFUser', null=True, on_delete=models.CASCADE)
+    user = models.ForeignKey('OSFUser', db_index=True, null=True, blank=True, on_delete=models.CASCADE)
+    params = DateTimeAwareJSONField(default=dict)
 
     SUBMITTED = 'submitted'
     REGISTERED = 'registered'
     APPROVED = 'approved'
     REJECTED = 'rejected'
 
+    EDITED_TITLE = 'edit_title'
+    EDITED_DESCRIPTION = 'edit_description'
+    CATEGORY_UPDATED = 'category_updated'
+
+    CONTRIB_ADDED = 'contributor_added'
+    CONTRIB_REMOVED = 'contributor_removed'
+    CONTRIB_REORDERED = 'contributors_reordered'
+    PERMISSIONS_UPDATED = 'permissions_updated'
+
+    MADE_CONTRIBUTOR_VISIBLE = 'made_contributor_visible'
+    MADE_CONTRIBUTOR_INVISIBLE = 'made_contributor_invisible'
+
+    AFFILIATED_INSTITUTION_ADDED = 'affiliated_institution_added'
+    AFFILIATED_INSTITUTION_REMOVED = 'affiliated_institution_removed'
+
+    CHANGED_LICENSE = 'license_changed'
+
+    TAG_ADDED = 'tag_added'
+    TAG_REMOVED = 'tag_removed'
```

Comment:
increases logging actions on this model as logs of editable field mixins already have logging capability and easy to add here. Need to verify with product that we do want these things logged. Practically, they won't be surfaced, but could be used for debugging? Also, would it be useful to copy these logs to the DraftNode after its converted to a node?